crates/core/tedge/src/cli/diag/collect.rs

```diff
@@ -200,7 +200,8 @@ impl DiagCollectCommand {
         tar.finish()?;
 
         // Cannot write this message to summary.log since the tarball has already been created
-        eprintln!("Diagnostic information saved to {tarball_path}");
+        eprint!("Diagnostic information saved to: ");
+        println!("{tarball_path}");
```

Comment:
Splitting a message over two channels is a bit unusual!

I would simply duplicate the message on each channel, serving two specific purposes (user feedback, script integration).

```suggestion
        eprintln!("Diagnostic information saved to {tarball_path}");
        println!("{tarball_path}");
```

Reply:
And worst case if people complain about that we can always add a conditional to only print the stdout message if the output is not a terminal

Reply:
Applied in a68a83b48df828de37d37c36a246ebeb4a8032c2